Assets/HoloToolkit-Preview/QRTracker/Scripts/AttachToQRCode.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using QRCodesTrackerPlugin;
+
+namespace HoloToolkit.Unity.QRTracking
+{
+    public class AttachToQRCode : MonoBehaviour
+    {
+        [Tooltip("Object that need to be attached to the QR code.")]
+        public GameObject gameObjectToAttach;
```

Comment:
private fields with property accessors.

Will we be able to update this game object after we've attached?
If we set a new game object, do we need to do any additional processing?

Reply:
not 100% required for a preview feature in the HTK